crates/db_views/src/comment_view.rs

```diff
@@ -514,14 +514,14 @@ impl ViewToVec for CommentView {
   type DbTuple = CommentViewTuple;
   fn from_tuple_to_vec(items: Vec<Self::DbTuple>) -> Vec<Self> {
     items
-      .iter()
+      .into_iter()
       .map(|a| Self {
-        comment: a.0.to_owned(),
-        creator: a.1.to_owned(),
-        recipient: a.3.to_owned(),
-        post: a.4.to_owned(),
-        community: a.5.to_owned(),
-        counts: a.6.to_owned(),
+        comment: a.0,
+        creator: a.1,
+        recipient: a.3,
+        post: a.4,
+        community: a.5,
+        counts: a.6,
```

Comment:
@dessalines You should use into_iter whenever possible, as that will take the value instead of reference, and allow you to avoid all these unnecessary clones. I already fixed this in all the db views.

Its also pretty annoying that this conversion method has to be copy-pasted all over the place, but there seems to be no way to avoid it as diesel can only load these queries into tuples and not structs.